### PR TITLE
OP-1057 - Update SQL scripts for OH version upgrade

### PR DESCRIPTION
**Pull request by mizzioisf**
See https://openhospital.atlassian.net/browse/OP-1057

### CODE REVIEW

File: sql/extra/README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
 # Purpose of this folder
 
-The scripts contained in this folder are temporary and for experimental features.
-
-They are not meant to be official release and they can be tested at your own risk.
\ No newline at end of file
+The scripts contained in this folder are for testing / experimental features.
+They are not meant to be used in production environment, use it at your own risk.
```
[the remark this answers is not in the document]
Comment (mizzioisf) [reply sql/extra/README.md]: Fixed, thank you!

File: sql/update/update_1_12_0-1_12_1.sql
Comment (mwithi) [inline sql/update/update_1_12_0-1_12_1.sql:1]: minor versions are not supposed to include changes to the DB (this file can be removed so far)

**Review by dbmalkovsky — APPROVED** [review]
LGTM

File: sql/update/update_1_12_0-1_12_1.sql
Comment (mizzioisf) [reply sql/update/update_1_12_0-1_12_1.sql:1]: done

File: sql/step_90_login_retries.sql
Comment (mwithi) [inline sql/step_90_login_retries.sql:1]: please restore this file

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**